althea_kernel_interface/src/link_local_tools.rs

```diff
@@ -75,6 +76,23 @@ impl dyn KernelInterface {
         }
     }
 
+    pub fn get_link_device_ip_v6(&self, dev: &str) -> Result<IpNetwork, KernelInterfaceError> {
```

Comment:
I believe this is a duplicate of get_link_local_device_ip at the top of this file but with an IpNetwork response. Maybe modify that function to return an IpNetwork?

Reply:
fixed